### ISSUE TITLE
quickfiles example: jinja2.exceptionsUndefinedError when trying to a add a file

### ISSUE THREAD

**Issue by timrichardson** [dpgaspar/Flask-AppBuilder#334]
Trying to add the first file to a project: 

https://gist.github.com/anonymous/bb6089f1afdd299c658af2b061d9e21c

`Traceback (most recent call last):
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/security/decorators.py", line 26, in wraps
    return f(self, *args, **kwargs)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/views.py", line 482, in add
    widgets=widget)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/baseviews.py", line 158, in render_template
    return render_template(template, **dict(list(kwargs.items()) + list(self.extra_args.items())))
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/home/tim/flask-env/lib/python3.5/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/tim/flask-env/lib/python3.5/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tim/flask-env/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/general/model/add.html", line 2, in top-level template code
    {% import 'appbuilder/general/lib.html' as lib %}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/base.html", line 1, in top-level template code
    {% extends base_template %}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/baselayout.html", line 2, in top-level template code
    {% import 'appbuilder/baselib.html' as baselib %}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/init.html", line 46, in top-level template code
    {% block body %}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/baselayout.html", line 19, in block "body"
    {% block content %}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/general/model/add.html", line 7, in block "content"
    {% block add_form %}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/general/model/add.html", line 8, in block "add_form"
    {{ widgets.get('add')(form_action=form_action)|safe }}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/widgets.py", line 34, in __call__
    return template.render(args)
  File "/home/tim/flask-env/lib/python3.5/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/tim/flask-env/lib/python3.5/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tim/flask-env/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/general/widgets/form.html", line 45, in top-level template code
    {{ lib.render_field(field, begin_sep_label, end_sep_label, begin_sep_field, end_sep_field) }}
  File "/home/tim/flask-env/lib/python3.5/site-packages/flask_appbuilder/templates/appbuilder/general/lib.html", line 195, in template
    {{ field.label.text }}
  File "/home/tim/flask-env/lib/python3.5/site-packages/jinja2/environment.py", line 408, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'None' has no attribute 'label'`

**Comment by timrichardson** [comment]
Also, these messages from the console log:

```
2016-06-12 12:39:00,204:WARNING:flask.ext.appbuilder.models.filters:Filter type not supported for column: file
2016-06-12 12:39:00,205:WARNING:flask.ext.appbuilder.models.filters:Filter type not supported for column: file
2016-06-12 12:39:00,205:ERROR:flask.ext.appbuilder.forms:Column file Type not supported
2016-06-12 12:39:00,205:ERROR:flask.ext.appbuilder.forms:Column file Type not supported
2016-06-12 12:39:00,205:ERROR:flask.ext.appbuilder.forms:Column file Type not supported
```

**Comment by timrichardson** [comment]
I have been trying to debug this to be more useful, but the learning curve ...

**Comment by dpgaspar** [comment]
I think that it might be it, i only support the following python versions:

2.6.X
2.7.X
3.3.X

Sorry not 3.5.X yet...

**Comment by dpgaspar** [comment]
I'll close this one, any further issues or questions, feel free to reopen it or open a new one.

**Comment by timrichardson** [comment]
Have you got any hints where I should look for a fix? Ubuntu LTS is 3.5 now and everything else is working ...
